discovery-provider/Caddyfile

```diff
@@ -1,6 +1,6 @@
 {$audius_discprov_url}
 
-{$CADDY_TLS}
+{$CADDY_TLS:localhost}
```

Comment:
this is how I was able to hit caddy locally, I can remove it prior to merging if we think it'll be problematic

Reply:
This breaks prod node deployment with
```
{"level":"info","ts":1703089907.6094224,"msg":"using provided configuration","config_file":"/etc/caddy/Caddyfile","config_adapter":"caddyfile"}
Error: adapting config using caddyfile: /etc/caddy/Caddyfile:3: unrecognized directive: localhost
```

Not sure why.

Reply:
Oh this makes sense actually - it's because this is not set by default and unless it's set, we will use the localhost default

https://caddyserver.com/docs/quick-starts/caddyfile

localhost only makes sense in a dev setting if not trying to expose publicly. so i think this should be removed but if you want this behavior locally 

export CADDY_TLS=localhost

would be the answer. 

remove this change and then good to go!!!

Reply:
oh nice thanks! will do